Gateway/Request/AddressDataBuilder.php

```diff
@@ -63,16 +63,23 @@ public function build(array $buildSubject)
         if ($billingAddress) {
 
             $requestBilling = ["street" => "N/A",
-                "postalCode" => '',
+                "postalCode" => 'N/A',
                 "city" => "N/A",
-                "houseNumberOrName" => '',
-                "stateOrProvince" => '',
+                "houseNumberOrName" => 'N/A',
+                "stateOrProvince" => 'N/A',
                 "country" => "ZZ"
             ];
 
-            if ($billingAddress->getStreetLine1()) {
-                $requestBilling["street"] = $billingAddress->getStreetLine1();
-            }
+			if ($billingAddress->getStreetLine1()) {
+				$address = $this->adyenHelper->getStreet($billingAddress->getStreetLine1());
```

Comment:
adyenHelper->getStreet() wants an address as a parameter, it does not work with string